package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "tree-sitter-proto",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "description": "A tree-sitter grammar for protocol buffers (proto3)",
   "main": "bindings/node",
   "types": "bindings/node",
   "scripts": {
-    "build": "tree-sitter generate && node-gyp build",
-    "test": "tree-sitter test",
-    "test-windows": "tree-sitter test",
     "install": "node-gyp-build",
-    "prebuildify": "prebuildify --napi --strip"
+    "build": "npx tree-sitter-cli generate --no-bindings",
+    "prestart": "npx tree-sitter-cli build --wasm",
+    "start": "npx tree-sitter-cli playground",
+    "test": "node --test bindings/node/*_test.js"
```

Comment:
Calling through npx ensures we use the correct version from the dev dependencies.

Reply:
Apparently this is not necessary. NPM ensures the binary from the dependency is on the path.